tutorials/time-freq/plot_ssvep.py

```diff
@@ -0,0 +1,698 @@
+# our frequency tagged response.
+#
+
+# find index of frequency bin closest to stimulation frequency
+i_bin_12hz = np.argmin(abs(np.subtract(freqs, stim_freq)))
+# could be updated to support multiple frequencies
+
+# for later, we will already find the 15 hz bin and the 1st and 2nd harmonic
+# for both.
+i_bin_24hz = np.argmin(abs(np.subtract(freqs, 24)))
+i_bin_36hz = np.argmin(abs(np.subtract(freqs, 36)))
+i_bin_15hz = np.argmin(abs(np.subtract(freqs, 15)))
+i_bin_30hz = np.argmin(abs(np.subtract(freqs, 30)))
+i_bin_45hz = np.argmin(abs(np.subtract(freqs, 45)))
+
+###############################################################################
+# Get indices for the different trial types
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+i_trial_12hz = np.where(epochs.events[:, 2] == event_id['12hz'])
+i_trial_15hz = np.where(epochs.events[:, 2] == event_id['15hz'])
+
+###############################################################################
+# Get indices for the EEG channels forming the ROI
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+# Define different ROIs
+roi_vis = ['POz', 'Oz', 'O1', 'O2', 'PO3', 'PO4', 'PO7',
+           'PO8', 'PO9', 'PO10', 'O9', 'O10']  # visual roi
+
+# Find corresponding indices using mne.pick_types()
+picks_roi_vis = mne.pick_types(epochs.info, eeg=True, stim=False,
+                               exclude='bads', selection=roi_vis)
+
+###############################################################################
+# Apply the subset, and check the result
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Now we simply need to apply our selection and yield a result. Therefore,
+# we typically report grand average SNR over the subselection.
+#
+
+snrs_target = snrs[i_trial_12hz, :, i_bin_12hz][0][:, picks_roi_vis]
+print("sub 2, 12hz trials, SNR at 12hz")
+print('average SNR (occipital ROI): %f' % snrs_target.mean())
+
+##############################################################################
+# Topography of the vSSR
+# ----------------------
+# But wait...
+# As described in the intro, we have decided *a priori* to work with average
+# SNR over a subset of occipital channels - a visual region of interest (ROI)
+# - because we expect SNR to be higher on these channels than in other
+# channels.
+#
+# Let's check out, whether this was a good decision!
+#
+# Here we will plot average SNR for each channel location as a topoplot.
+# Then we will do a simple paired T-test to check, whether average SNRs over
+# the two sets of channels are significantly different.
+#
+
+# get average SNR at 12hz for ALL channels
+snrs_12hz = snrs[i_trial_12hz, :, i_bin_12hz][0]
+snrs_12hz_chaverage = snrs_12hz.mean(axis=0)
+
+# create a standard montage
+montage = mne.channels.make_standard_montage('easycap-M1', head_size=0.095)
+
+# add xyz coordinates for all channels
+montage.positions = montage._get_ch_pos()
```

Comment:
you cannot use private functions / methods in a public tutorial

Reply:
I guess there is a non-private way to get xyz coordinates of channels for use with `viz.plot_topomap`, but i didnt find it. could you give me a hint?

Reply:
Can you avoid getting the positions yourself and instead use the `mask` parameter of `plot_topomap`?

Reply:
ok, i just realized that i can pass an `info` object as position argument.. much easier